addons/mail/wizard/mail_compose_message.py

```diff
@@ -101,55 +112,62 @@ def _get_composition_mode_selection(self):
                 ('mass_mail', 'Email Mass Mailing'),
                 ('mass_post', 'Post on Multiple Documents')]
 
+    # content
+    subject = fields.Char('Subject')
+    body = fields.Html('Contents', default='', sanitize_style=True)
+    parent_id = fields.Many2one(
+        'mail.message', 'Parent Message', index=True, ondelete='set null',
+        help="Initial thread message.")
+    attachment_ids = fields.Many2many(
+        'ir.attachment', 'mail_compose_message_ir_attachments_rel',
+        'wizard_id', 'attachment_id', 'Attachments')
+    # origin
+    email_from = fields.Char('From', help="Email address of the sender. This field is set when no matching partner is found and replaces the author_id field in the chatter.")
+    author_id = fields.Many2one(
+        'res.partner', 'Author', index=True,
+        help="Author of the message. If not set, email_from may hold an email address that did not match any partner.")
+    # related document
+    model = fields.Char('Related Document Model', index=True)
+    res_id = fields.Integer('Related Document ID', index=True)
+    record_name = fields.Char('Message Record Name', help="Name get of the related document.")
+    # characteristics
+    message_type = fields.Selection([
+        ('comment', 'Comment'),
+        ('notification', 'System notification')],
+        'Type', required=True, default='comment',
+        help="Message type: email for email message, notification for system "
+             "message, comment for other messages such as user replies",
+        oldname='type')
```

Comment:
Why can't we use `email` as message_type using the wizard anymore (in 13.0)?

And the `help` message is not correct.

Reply:
Because email is used for incoming emails.

About help, feel free to open a PR with your proposal.